1-org/envs/shared/org_policy.tf

```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${var.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
```

Comment:
documentation says the value should be `INTERNAL`
https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints#:~:text=The%20list%20of%20allowed%20or%20denied%20values%20can,EXTERNAL

does it work with both `INTERNAL` and `internal` ?

Reply:
It did not raise any error message so far, but I'll change it to capital anyway.